config.json

```diff
@@ -29,1735 +29,1735 @@
         "version": "16.1.0",
-        "nugetVersion": "116.1.0.1",
+        "nugetVersion": "116.1.0.2",
         "nugetId": "Xamarin.GooglePlayServices.Code.Scanner",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cronet",
         "version": "18.0.1",
-        "nugetVersion": "118.0.1.6",
+        "nugetVersion": "118.0.1.7",
         "nugetId": "Xamarin.GooglePlayServices.CroNet",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-deviceperformance",
         "version": "16.0.0",
-        "nugetVersion": "116.0.0.2",
+        "nugetVersion": "116.0.0.3",
         "nugetId": "Xamarin.GooglePlayServices.DevicePerformance",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-drive",
         "version": "17.0.0",
-        "nugetVersion": "117.0.0.12",
+        "nugetVersion": "117.0.0.13",
         "nugetId": "Xamarin.GooglePlayServices.Drive",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-fido",
         "version": "20.0.1",
-        "nugetVersion": "120.0.1.3",
+        "nugetVersion": "120.0.1.4",
         "nugetId": "Xamarin.GooglePlayServices.Fido",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-fitness",
         "version": "21.1.0",
-        "nugetVersion": "121.1.0.5",
+        "nugetVersion": "121.1.0.6",
         "nugetId": "Xamarin.GooglePlayServices.Fitness",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-flags",
         "version": "18.0.1",
-        "nugetVersion": "118.0.1.9",
+        "nugetVersion": "118.0.1.10",
         "nugetId": "Xamarin.GooglePlayServices.Flags",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-games",
         "version": "23.1.0",
-        "nugetVersion": "123.1.0.4",
+        "nugetVersion": "123.1.0.5",
         "nugetId": "Xamarin.GooglePlayServices.Games",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-games-v2",
         "version": "19.0.0",
-        "nugetVersion": "119.0.0",
+        "nugetVersion": "119.0.0.1",
         "nugetId": "Xamarin.GooglePlayServices.Games.V2",
         "dependencyOnly": false
       },
-      {
-        "groupId": "com.google.android.gms",
-        "artifactId": "play-services-gass",
-        "version": "20.0.0",
-        "nugetVersion": "120.0.0.11",
-        "nugetId": "Xamarin.GooglePlayServices.Gass",
-        "dependencyOnly": false
-      },
+      // {
```

Comment:
Why are we commenting out these 3 packages?

Reply:
When updating I comment out those 3 at the beginning, because when there are more updates and bigger gaps in versions and dependencies with pinned/fixed versions then sometimes it is difficult to match all versions.

Besides those are very old packages which were not updated for a long time and additionally have pinned/exact versions (`[x.y.z]`) which makes tuning binderator problematic. Especially if the version jumps/bumps are bigger (bindings not updated for longer time)

1. `com.google.android.gms.play-services-gass-20.0.0` 
     not updated since 2021. 
2. `com.google.firebase.firebase-analytics-impl-16.3.0`
     not updated since 2019.
3. `com.google.android.gms.play-services-measurement-base:16.3.0`
     dependency only
     not updated since 2019.

`com.google.android.gms.play-services-gass-20.0.0` since 2021.

```xml
<?xml version='1.0' encoding='UTF-8'?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.google.android.gms</groupId>
  <artifactId>play-services-gass</artifactId>
  <version>20.0.0</version>
  <packaging>aar</packaging>
  <dependencies>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-ads-base</artifactId>
      <version>[20.0.0]</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-ads-identifier</artifactId>
      <version>17.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-ads-lite</artifactId>
      <version>[20.0.0]</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-basement</artifactId>
      <version>17.6.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-tasks</artifactId>
      <version>17.1.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
  </dependencies>
  <name>play-services-gass</name>
  <licenses>
    <license>
      <name>Android Software Development Kit License</name>
      <url>https://developer.android.com/studio/terms.html</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
</project>
```

`com.google.firebase.firebase-analytics-impl-16.3.0`since 2019.:

```xml
<?xml version='1.0' encoding='UTF-8'?>
<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.google.firebase</groupId>
  <artifactId>firebase-analytics-impl</artifactId>
  <version>16.3.0</version>
  <packaging>aar</packaging>
  <dependencies>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-ads-identifier</artifactId>
      <version>16.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-basement</artifactId>
      <version>16.0.1</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-measurement-base</artifactId>
      <version>[16.3.0]</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-stats</artifactId>
      <version>16.0.1</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-tasks</artifactId>
      <version>16.0.1</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-common</artifactId>
      <version>16.0.3</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-iid</artifactId>
      <version>17.0.3</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
  </dependencies>
  <name>firebase-analytics-impl</name>
  <licenses>
    <license>
      <name>Android Software Development Kit License</name>
      <url>https://developer.android.com/studio/terms.html</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
</project>
```